src/appshell/qml/DevTools/MPE/ArticulationsProfileEditorView.qml

```diff
@@ -63,7 +63,7 @@ Rectangle {
 
                     width: parent.width - openButton.width - saveButton.width - createButton.width
 
-                    hint: qsTrc("mpe", "Profile path")
+                    hint: "Profile path" //qsTrc("mpe", "Profile path")
```

Comment:
I think, we can remove comments

Reply:
I decided to keep them, but only in a simpler form, to make them less obstructive. When we are going to expose this tool to users, we can restore translatability with a simple RegEx search & replace, using these comments.